cmscontrib/loaders/tps.py

```diff
@@ -123,6 +124,7 @@ def get_task(self, get_statement=True):
 
         args["name"] = name
         args["title"] = data['name']
+        args["score_mode"] = SCORE_MODE_MAX_SUBTASK
```

Comment:
Cannot find existing tests for TPS task loader, so I guess it is fine without test 😅